benchmarks/mwt/README.md

```diff
@@ -0,0 +1,48 @@
+
+1. assert kudo 
+    k kudo init --wait
+    k kudo init --wait --unsafe-self-signed-webhook-ca
+
+2. create ns
+2. install CR
+    assert
+3. install with params
+    with instance name
+    assert
+
+## Prerequisites
+
+* Kubernetes cluster up
+* KUDO CLI installed on node kuttl is running from
+* KUDO manager installed in the cluster
+
+
+# TestSuites
+
+* **Setup:** used to setup a MWT test use the following command 
+
+`kubectl kuttl test setup/  --parallel 1 --skip-delete`
```

Comment:
I'd like to see a kuttl TestSuite file that has these parameters. Having them as parameters and to remember to add them seems like an easy cause of failure

Reply:
that is easily done... however that would require a kuttl-test.yaml for 2+ sets of tests... the setup and the teardown... which means need to run `kubectl kuttl test --config kuttl-setup.yaml` and  `kubectl kuttl test --config kuttl-teardown.yaml`

So there are still things to "know"... but it does push out config details... (which I like a lot).  I'll add this in.